<launch>
	<arg name="record_pose" value="false"/>

	<rosparam command="load" file="$(find local_planner)/params/cfg.yaml"/>
	<node pkg="tf" type="static_transform_publisher" name="footprint_broadcaster" args="0 0 0 0 0 0 1 base_link footprint 100" />
	<node pkg="tf" type="static_transform_publisher" name="search_broadcaster" args="0 0 0 0 0 0 1 base_link search_area 100" />
	<node name="costmap_clear_node" type="costmap_clear_node" pkg="local_planner" output="screen"/>
	<node name="local_planner_node" type="local_planner_node" pkg="local_planner" output="screen"/>

	<group if="$(arg record_pose)">
		<include file="$(find local_planner)/launch/record_pose.launch"/>
	</group>
</launch>	
